test/new_relic/agent/configuration/event_harvest_config_test.rb

```diff
@@ -33,7 +35,8 @@ def test_to_config_hash
           'harvest_limits' => {
             'analytic_event_data' => 833,
             'custom_event_data' => 83,
-            'error_event_data' => 8
+            'error_event_data' => 8,
+            'log_event_data' => 833
```

Comment:
Is this the number of spans per faster event harvest (5 seconds)? 833 * 12 = 9996?

Reply:
Yeah, that's per 5 second harvest. 10,000/12 = 833.3333 is where that's coming from.